client/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces.ts

```diff
@@ -0,0 +1,71 @@
+import { ReactNode } from 'react';
+
+export interface ItemsDataViewsType< T > {
+	items: T[] | undefined;
+	pagination: DataViewsPaginationInfo;
+	searchLabel?: string;
+	fields: DataViewsColumn[];
+	actions?: DataViewsAction[];
+	getItemId?: ( item: T ) => string;
+	itemFieldId?: string; // The field path to get the item id. Examples `id` or `site.blog_id`
```

Comment:
**New field:** with this field we can point where is the id field. DataViews tries to find the item.id, but not all the elements have their id in this form. For instance, the Site element we use at A4A the `id` is here: `site.value.blog_id`.